src/ipyradiant/visualization/interactive_exploration.py

```diff
@@ -0,0 +1,309 @@
+# Copyright (c) 2021 ipyradiant contributors.
+# Distributed under the terms of the Modified BSD License.
+
+import traitlets as trt
+
+import ipycytoscape
```

Comment:
small thing, but maybe:

`import ipycytoscape as cyto`

so the usages below are a bit more succinct...

Reply:
or something else... e.g., `ipycy`